### PR TITLE
add login and register with tests

### PR DESCRIPTION
**Pull request by jinpark**
Adds login and register with tests

Still more work needs to be done but this is in a "working" state

Ref: #3 1st part in checklist

### CODE REVIEW

File: accounts/templates/registration/register.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+{% load staticfiles %}
+{% load form_tags %}
+
+{% block content %}
+    <div class="mdl-grid">
+      <div class="mdl-cell mdl-cell--4-col"></div>
+      <div class="mdl-cell mdl-cell--4-col">
+        <h2>register</h2>
+        {% if form.errors %}
+        <p>Something went wrong. Blame jin.</p>
```
Comment (hassanabidpk) [inline accounts/templates/registration/register.html]: lol

File: code4every1site/templates/base.html [hunk 2]
```diff
@@ -109,6 +110,14 @@
               <a class="mdl-navigation__link mdl-typography--text-uppercase" href="/hacks/">Hacks</a>
               <a class="mdl-navigation__link mdl-typography--text-uppercase" href="/about-us/">About us</a>
               <a class="mdl-navigation__link mdl-typography--text-uppercase" href="/contact-us/">Contact us</a>
+              {% if request.user.is_anonymous %}
+                <a class="mdl-navigation__link mdl-typography--text-uppercase" href="/accounts/login">Login</a>
+                <a class="mdl-navigation__link mdl-typography--text-uppercase" href="/accounts/register">Register</a>
+              {% endif %}
+              {% if not request.user.is_anonymous %}
+                <a class="mdl-navigation__link mdl-typography--text-uppercase" href="/accounts/login">{{ request.user }}</a>
```
Comment (hassanabidpk) [inline code4every1site/templates/base.html]: We will be changing this to a my profile page?  Where they can set First and Last Name etc.

Comment (jinpark) [inline code4every1site/templates/base.html]: Yea, I'll add this to the next pr

File: accounts/templates/registration/register.html (hunk 1 above)
Comment (jinpark) [inline accounts/templates/registration/register.html]: lol, let me make this into real errors

**Comment by hassanabidpk** [comment]
It looks vey neat. I went through it line by line and tested it on my computer.

**Pull request merged**